test/createAzureResource.test.ts

```diff
@@ -129,3 +162,38 @@ function getWebsiteManagementClient(testAccount: TestAzureAccount): WebSiteManag
 function getResourceManagementClient(testAccount: TestAzureAccount): ResourceManagementClient {
     return new ResourceManagementClient(testAccount.getSubscriptionCredentials(), testAccount.getSubscriptionId());
 }
+
+// The root workspace ".zip" that vscode is opened against for tests
+function getTestRootZipFile(): string {
```

Comment:
Could you make a general command for finding the workspace like this?
https://github.com/microsoft/vscode-azurefunctions/blob/master/test/global.test.ts#L102-L114

And then another function that will get the zip file?